offchain-modules/lib/src/util/eth_util.rs

```diff
@@ -140,6 +155,16 @@ fn rlp_append<TX>(header: &Block<TX>, stream: &mut RlpStream) {
     stream.append(&header.nonce.unwrap());
 }
 
+pub fn convert_eth_address(mut address: &str) -> Result<H160> {
+    if address.starts_with("0x") || address.starts_with("0X") {
+        address = &address[2..];
+    }
+    if address.len() != ETH_ADDRESS_LENGTH {
+        return Err(Error::msg(format!("invalid eth address: {:?}", address)));
```

Comment:
```suggestion
        anyhow::bail!("invalid eth address: {:?}", address);
```

Simpler way for this.

Reply:
copy that